program/globals.cpp

```diff
@@ -33,6 +33,7 @@ double g_absoluteZoom;
 QSharedPointer<BlastSearch> g_blastSearch;
 QString m_tempDirectory;
 QSharedPointer<AssemblyGraph> g_assemblyGraph;
+std::shared_ptr<AnnotationsManager> g_annotationsManager;
```

Comment:
Do we really need to have it shared_ptr here?

Reply:
unique_ptr need a complete type here. I can't understand why...